core/src/test/kotlin/OAuthAppExample.kt

```diff
@@ -33,7 +33,7 @@ class StateManager {
   }
 }
 
-fun urlEncode(v: String?): String = URLEncoder.encode(v, Charsets.UTF_8)
+fun urlEncode(v: String?): String = URLEncoder.encode(v, "UTF-8")
```

Comment:
Why did you change this?

Reply:
When I was initially compiling the project, I was using JDK 8.  I was getting the following error:

```
[ERROR] /.../eblanchette/notion-sdk-jvm/core/src/test/kotlin/OAuthAppExample.kt:[36,58] Type mismatch: inferred type is Charset but String! was expected
```

I have since updated to JDK 11 and the error goes away.  I will revert my changes.